src/infra/pg-secrets.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: pg-secrets
+type: Opaque
+data:
+  POSTGRES_USER: cG9zdGdyZXM=
+  POSTGRES_PASSWORD: cm9vdA==
+  DATASOURCE_URL: amRiYzpwb3N0Z3Jlc3FsOi8vcG9zdGdyZXM6NTQzMi9wb3N0Z3Jlcw==
+  DATASOURCE_USERNAME: cG9zdGdyZXM=
+  DATASOURCE_PASSWORD: cm9vdA==
+  SCHEMA: aG9zcGl0YWw=
```

Comment:
No need to encrypt schema information. Remember it's all about maintainability and traceability of the project!